### ISSUE TITLE
why noExport default options set true..?

### ISSUE THREAD

**Issue by youngrove** [cometkim/gatsby-plugin-typegen#170]
Issue 
- useStaticQuery auto fixed but default export options set noExport.

My auto generated types is here..
```
// no export 
declare namespace GatsbyTypes {
...
```

But, I can't use GatsbyTypes for My query.
<img width="1080" alt="Screen Shot 2022-01-30 at 7 24 16 PM" src="https://user-images.githubusercontent.com/40445771/151695844-9d8787e2-7945-47d7-9bc9-f27bc3c2af11.png">

In codegen.ts 
```
const DEFAULT_TYPESCRIPT_CONFIG: Readonly<TypeScriptPluginConfig> = {
  avoidOptionals: true,
  immutableTypes: true,
  maybeValue: 'T | undefined',
  noExport: true,
  enumsAsTypes: true,
};
```

**Comment by youngrove** [comment]
I added this line when I used generated types...
```
/// <reference path = "../__generated__/gatsby-types.ts" />
```
And I Got this error. I Knew If I change ts.config file It will resolved.. 
But I wonder why set default noexport options set true..

'gatsby-types.ts' cannot be compiled under '--isolatedModules' because it is considered a global script file. Add an import, export, or an empty 'export {}' statement to make it a module.
